### DISCUSSION TITLE
定时任务

### DISCUSSION THREAD

**Discussion by Sjxixi-y** [simple-robot/Discussions#168] [Q&A]
### 涉及的编程语言

Java

### 涉及的组件库

mirai (QQ)

### 疑问描述

如何在不触发监听器的情况下获取指定的群对象，上次  `Mirai.getInstance()` 中没有类似的方法。或者只能发个消息让他保存在Map里？

**Comment by ForteScarlet** [comment]
参考 #74 ，意思差不多，能拿到 `Bot` 就能拿到其他的。
